bundle.md

```diff
@@ -8,17 +8,21 @@ See also [MacOS application bundles][macos_bundle] for a similar use of the term
 The definition of a bundle is only concerned with how a container, and its configuration data, are stored on a local filesystem so that it can be consumed by a compliant runtime.
 
 A Standard Container bundle contains all the information needed to load and run a container.
-This MUST include the following artifacts:
+This includes the following artifacts:
 
 1. <a name="containerFormat01" />`config.json`: contains configuration data.
 This REQUIRED file MUST reside in the root of the bundle directory and MUST be named `config.json`.
 See [`config.json`](config.md) for more details.
 
 2. <a name="containerFormat02" />A directory representing the root filesystem of the container.
-While the name of this REQUIRED directory may be arbitrary, users should consider using a conventional name, such as `rootfs`.
-This directory MUST be referenced by [`root`](config.md#root) within the `config.json` file.
+    While the name of this directory may be arbitrary, users should consider using a conventional name, such as `rootfs`.
 
-While these artifacts MUST all be present in a single directory on the local filesystem, that directory itself is not part of the bundle.
-In other words, a tar archive of a *bundle* will have these artifacts at the root of the archive, not nested within a top-level directory.
+    On Windows, for Windows Server containers, this directory is REQUIRED. For Hyper-V containers, it MUST be omitted.
+
+    On all other platforms, this field is REQUIRED.
+
+    If set, this directory MUST be referenced by [`root`](config.md#root) within the `config.json` file.
+
+When supplied, while these artifacts MUST all be present in a single directory on the local filesystem, that directory itself is not part of the bundle. In other words, a tar archive of a *bundle* will have these artifacts at the root of the archive, not nested within a top-level directory.
```

Comment:
You seem to be removing the newline before “In other words” while not touching the content of that sentence.  I think you want to [keep that sentence on its own line][1].

Also, “When supplied, while …” is a bit awkward.  I keep circling back to #469 when trying to think up clearer phrasing, so I don't have anything new to propose as an alternative.  But probably something that separates out the same-directory MUST from the definition of “bundle” as the parent of `config.json` (and any other artifacts listed here).

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc5/style.md#one-sentence-per-line